utils/setools/patches/010-no-gui.patch

```diff
@@ -1,16 +1,19 @@
 --- a/setup.py
 +++ b/setup.py
```

Comment:
I wonder, do we need to have this patch at all? It looks liek you are removing setoolsgui. 

- There isnt any configuration option to disable it?
- You can not remove by using e.g. filespec ?
- GUI can be used for x86 target, isnt it?